zha/application/platforms/sensor/__init__.py

```diff
@@ -475,6 +476,37 @@ def formatter(self, value: int) -> str | None:
         assert self._enum is not None
         return self._enum(value).name
 
+@CONFIG_DIAGNOSTIC_MATCH(cluster_handler_names=CLUSTER_HANDLER_MULTISTATE_INPUT)
+class MultiStateInputSensor(EnumSensor):
+    """Sensor that displays enumerated values."""
+
+    _attribute_name = "present_value"
+    _unique_id_suffix = "present_value"
+    _attr_translation_key: str = "multistate_input"
+    _attr_entity_registry_enabled_default = False
+    _attr_extra_state_attribute_names: set[str] = {
```

Comment:
`_attr_extra_state_attribute_names` are going away and should be converted into entities themselves, if they're necessary.